sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/TypesafeBackendImpl.kt

```diff
@@ -27,6 +27,9 @@ internal class TypesafeBackendImpl(private val backend: Backend) : TypesafeBacke
     override val network: ZcashNetwork
         get() = ZcashNetwork.from(backend.networkId)
 
+    override suspend fun getAccounts(): List<Account> =
+        backend.getAccounts().map { Account(it.accountId.toInt()) }
```

Comment:
@str4d should we expose the `ufvk` from `JniAcocunt` if there is any? Or we plan to do it in the next Account logic iteration?

Reply:
I'm exposing it here because we already get it from the SDK, and passing it across the JNI now means that we don't have to do so later when you inevitably need it. But given that you don't need it right now, we should leave it alone inside `JniAccount`.